locales/en.default.schema.json

```diff
@@ -587,8 +636,29 @@
         "show_view_all": {
           "label": "Enable \"View all\" button if collection has more products than shown"
         },
-        "swipe_on_mobile": {
-          "label": "Enable swipe on mobile"
+        "view_all_style": {
+          "label": "View all style",
```

Comment:
@melissaperreault I'd agree "View all" in the label should not have quotation marks - I think this exists in a few other sections - I'll need to document and get them updated if we change here. 

If I understand your second question correctly, I opted to remove the mention of button because one of the styling options will be a link, rather than a button. Does that make sense?

Reply:
If we want to update the label I suggest changing it all at once in another issue if we decide to remove the quotation marks. I recommend making sure that this change would translate well to other languages.

For the second part, this is what I assumed. I find the label don't match other more neutral `style` settings. It's hard to quickly spot while scanning the list of settings where the button style is. I don't think this would translate well to other languages either.

i.e. Other neutral form: We have already `**Text** style`, `**Description** style`, and `Use button outline style`; `**View all** style` stood out to me.

Also, this section is the only one to wear the `link` button style by default, when a merchant adds the section among other section, its default button visual weight would be different, should it be `Button` instead since the carousel if set to `false` by default on desktop? cc. @YoannJailin @danielvan

Reply:
Hm I see what you mean with the label, Meli. Do you think it would be better to say `Button style` even if one of the options is a link? Maybe a more neutral approach could be `Call to action style`. I'll do some exploration, thanks for flagging. 

RE: default, if all other sections use button as default I agree that should be the case here too. Will be easier for merchants to achieve a consistent look.